pages/table/tall-rows.page.tsx

```diff
@@ -22,7 +22,7 @@ export default function () {
       ) : (
         <ul>
           {range(0, 20).map(index => {
-            return <li key={index}>{(config.cell as TableProps.CellRenderer<Instance>)(item)}</li>;
+            return <li key={index}>{config.cell(item)}</li>;
```

Comment:
Typecast no longer needed for ambiguous calls (previously typescript wasn't be able to infer the number of arguments)